### DISCUSSION TITLE
Documentation website

### DISCUSSION THREAD

**Discussion by jakoss** [s0nicyouth/kmapper#8] [Ideas]
Have you considered how do you want to host the documentation? I think docusaurus is great: https://docusaurus.io/ It's very easy to setup and deploy to GitHub Pages (https://docusaurus.io/docs/deployment#deploying-to-github-pages). I'd love to contribute docs, but we need some basic setup for this to start :)

**Comment by s0nicyouth** [comment]
Well, yeah definitely documentation is one of the priorities here. I will research and get back.
